packages/next-app/src/app/daoadmin/action/[actionId]/(components)/ActionAttributeContent.tsx

```diff
@@ -1,5 +1,6 @@
 "use client";
 
+import _ from "lodash";
```

Comment:
prefer importing only the function you'll use otherwise this is likely to load the full lib to the code
```suggestion
import isEmpty from "lodash/isEmpty";
```